dotcom-rendering/lighthouserc.js

```diff
@@ -1,7 +1,10 @@
 module.exports = {
   ci: {
     collect: {
-      url: ['http://localhost:9000/Article?url=https://www.theguardian.com/commentisfree/2020/feb/08/hungary-now-for-the-new-right-what-venezuela-once-was-for-the-left#noads'],
+      url: [
+        'http://localhost:9000/Article?url=https://www.theguardian.com/commentisfree/2020/feb/08/hungary-now-for-the-new-right-what-venezuela-once-was-for-the-left#noads',
+        'http://localhost:9000/Front?url=https://www.theguardian.com/uk'
```

Comment:
I thought we would start with international?

```suggestion
        'http://localhost:9000/Front?url=https://www.theguardian.com/international'
```